UserManagement/README.md

```diff
@@ -1,32 +0,0 @@
-### Software requirements

```

Comment:
Are these instructions not valid anymore ?

Reply:
They are valid. Just need to append instructions for Docker as well.